internal/cli/login.go

```diff
@@ -253,6 +253,12 @@ func (o *LoginOptions) Run(ctx context.Context, args []string) error {
 	if err != nil {
 		return fmt.Errorf("persisting client config: %w", err)
 	}
+
+	defaultConfigPath := client.DefaultFlightctlClientConfigPath()
```

Comment:
Ouch. I just noticed we're defining the client's config file in two locations (`internal/cli/global` and `internal/client`) and inconsistently.

@noga-magen could I please ask you to introduce a `client.DefaultFlightctlClientConfigDir()` based on `os.UserConfigDir()` and update both the `client.DefaultFlightctlClientConfigPath()` and the `ConfigDir()` to use it? Thanks.